helm/memcached-operator/README.md

```diff
@@ -99,9 +99,11 @@ This project was created by using the following command, which means that it is
 ```shell
 operator-sdk new memcached-operator --api-version=cache.example.com/v1alpha1 --kind=Memcached --type=helm --helm-chart=stable/memcached
 ```
+Note that for you are able to use the stable helm charts you need have helm installed as the stable repo added on it. For further information see the [Helm Quick Start][helm-quick].
```

Comment:
@camilamacedo86 can you make a follow-up PR with a grammar fix:
```
Note that you must add the stable helm charts repo to your cluster to use stable helm charts. See the [Helm Quick Start guide][helm-quick] for installation instructions.
``

Reply:
Hi @estroz, 

Tks for the suggestion. It was done here: https://github.com/operator-framework/operator-sdk-samples/pull/113

However, note that I did a few changes as well. See that the repo is not added to the cluster but it to/in the helm installed locally. Let me know wdyt?